ixnetwork_open_traffic_generator/trafficitem.py

```diff
@@ -551,14 +554,18 @@ def transmit(self, request):
         self._api._traffic_item.find(Name=regex)
         if len(self._api._traffic_item) > 0:
             if request.state == 'start':
+                self._api._traffic_item.find(Name=regex, State='^started$')
```

Comment:
Would State='^started$', Suspend=True find all items that paused?
Then there shouldn't be a need for a try?

Reply:
Thanks!